mitodl/mit-open
# update og-image with new logo

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/5727

### Description (What does it do?)
Updates our opengraph social preview image to use new logo.

### Screenshots (if appropriate):
Image should be 

![image](https://github.com/user-attachments/assets/7e156ddb-0a63-4cd7-a095-3dd7fa5cf222)


### How can this be tested?
1. Spin up this branch. Once the page has rendered, run `$$('meta').forEach(e => console.log(e.outerHTML))` in the JS dev console and check that both `<meta property="og:image" ... />` and `<meta name="twitter:image:src ... />` have an absolute URL pointing to the image above.

You can't really check the social preview locally, unfortunately, since it relies on prerender.io to actually wiork. (For now.)

### Checklist:
- [ ] We'll need to clear the prerender.io cache after deploy
